go/processor/geth/processor.go

```diff
@@ -318,13 +322,13 @@ func (s *stateDB) Empty(address common.Address) bool {
 }
 
 func (s *stateDB) AddressInAccessList(address common.Address) bool {
-	//lint:ignore SA1019 deprecated functions to be migrated
-	return s.context.IsAddressInAccessList(tosca.Address(address))
+	return bool(s.context.AccessAccount(tosca.Address(address)))
 }
 
 func (s *stateDB) SlotInAccessList(address common.Address, slot common.Hash) (addressOk bool, slotOk bool) {
-	//lint:ignore SA1019 deprecated functions to be migrated
-	return s.context.IsSlotInAccessList(tosca.Address(address), tosca.Key(slot))
+	addressOk = bool(s.context.AccessAccount(tosca.Address(address)))
+	slotOk = bool(s.context.AccessStorage(tosca.Address(address), tosca.Key(slot)))
```

Comment:
Same here, this implementation has unexpected side-effects.

I guess it is fine, since the `stateDB` is only intended for a single purpose, but one might have to explain this in a comment.